### DISCUSSION TITLE
Poetry run alias

### DISCUSSION THREAD

**Discussion by benhubsch** [python-poetry/poetry#7750] [Q&A]
I frequently run `poetry run mypy .` followed by `poetry run pflake8` and some other commands for linting. There's enough overhead in typing `poetry run` that I'd like to define an alias so that I can run `poetry run lints` to run `poetry run mypy . && poetry run pflake8` under the hood, for example.

Is there a way of doing this? I've wondered if I could somehow use the [scripts](https://python-poetry.org/docs/pyproject/#scripts) section to hack something in. I also know I could set a global command-line alias, but I want to define the alias within Poetry itself.

Thanks in advance!

**Comment by glencairn** [comment]
I suggest that you take a look at `poethepoet`: https://github.com/nat-n/poethepoet

It's a task runner that works well with `poetry`.
